### PR TITLE
added new analysis task for video-watching-time

### PR DESCRIPTION
**Pull request by ivan-abc**
New directory for analysis tasks created, an initial analysis task was put into the folder

### CODE REVIEW

File: analysis_tasks/get_video_watching_durations.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    date = dt[0].split("-")
+    time = dt[1].split(".")[0].split(":")
+    return datetime.datetime(int(date[0]), int(date[1]), int(date[2]), hour=int(time[0]), minute=int(time[1]), second=int(time[2]))
+
+
+# Prints total time of video-watching for every user (as a table)
+def print_result_table(user_times):
+    print("username | time (sec)")
+    for username in user_times:
+        print(username, "|", user_times[username].total_seconds())
```
Comment (mandarinSh) [inline analysis_tasks/get_video_watching_durations.py]: May be we should put results into .csv file? We've got special directory for that :-) ```libs/result/```

File: analysis_tasks/get_video_watching_durations.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    # Get all play and pause events from video_events table
+    cur.execute("SELECT event_type, username, time FROM video_events WHERE event_type='pause_video' OR event_type='play_video'")
+    # Get dict of users and their total video watching time values
+    # Time is represented as datetime.timedelta type
+    user_times = calculate_times_for_users(cur.fetchall())
+
+    # Print results of analytics task as table
+    print_result_table(user_times)
+
+    cur.close()
```
Comment (mandarinSh) [inline analysis_tasks/get_video_watching_durations.py]: Shouldn't we commit changes to DB to end transaction? ```con.commit()```

File: analysis_tasks/get_video_watching_durations.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    con = psycopg2.connect(
+      database=db_params['database'],
+      user=db_params['user'],
+      password=db_params['password'],
+      host=db_params['host'], 
+    )
+    cur = con.cursor()
+
+    # Get all play and pause events from video_events table
+    cur.execute("SELECT event_type, username, time FROM video_events WHERE event_type='pause_video' OR event_type='play_video'")
```
Comment (mandarinSh) [inline analysis_tasks/get_video_watching_durations.py]: Do we have such table? Or you'll create it later?

File: analysis_tasks/get_video_watching_durations.py (hunk 1 above)
Comment (ivan-abc) [reply analysis_tasks/get_video_watching_durations.py]: Ok, i haven't noticed that :)
So i'll add this feature

**Comment by IgorNikiforov** [comment]
Hello @ivan-abc ,
Thank you for your patch.
Could you please update the implementation in accordance to the latest structure of the project and create another one pull request.

**Pull request closed without merging**